src/main/java/nextstep/laddergame/domain/MemberLocations.java

```diff
@@ -0,0 +1,36 @@
+package nextstep.laddergame.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class MemberLocations {
+
+  private List<MemberLocation> memberLocations;
```

Comment:
```suggestion
  private final List<MemberLocation> memberLocations;
```

인스턴스 변수에 의식적으로 final을 먼저 추가하고, 값을 변경해야할 경우만 final을 제거하면 어떨까?